product_variant_multi_link/tests/test_product_template_link.py

```diff
@@ -3,10 +3,10 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo.exceptions import ValidationError
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class TestProductVariantLink(SavepointCase):
+class TestProductVariantLink(TransactionCase):
```

Comment:
You can inherit from `BaseCommon` class and remove the `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))` line in `setUpClass` method

Reply:
I updated it. Thanks :pray: